src/components/security_manager/CMakeLists.txt

```diff
@@ -55,10 +53,10 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
     string( REPLACE / \\ group ${inc} )
     source_group("Header Files" FILES ${inc} )
     add_library("SecurityManager" ${inc} ${SOURCES})
-    target_link_libraries(SecurityManager libeay32 ssleay32 ProtocolHandler jsoncpp ProtocolLibrary)
+    target_link_libraries(SecurityManager ${OPENSSL_LIBRARIES} protocol_handler jsoncpp protocol)
```

Comment:
@anosach-luxoft 
Maybe use LIBRARIES instead of "protocol_handler jsoncpp protocol"?